kratos/processes/compute_nodal_normal_divergence_process.cpp

```diff
@@ -0,0 +1,249 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//                   Vicente Mataix Ferrandiz
+//  Co-author:       Mohammad R. Hashemi
```

Comment:
aren't you the author?

Reply:
@philbucher thanks for reviewing.
"aren't you the author?" Yes, but I wanted to keep the track of the original ComputeNodalGradientProcess. Anyway, I was not sure about the KRATOS convention/rule. 

About using assertAlmostEqual:
The point is that the expected error is too large (comparing e.g. 3.7 with 4.0). I do not know what options do I have?

Reply:
regarding authorship I recommend sth like this:
https://github.com/KratosMultiphysics/Kratos/blob/d0e137e56bd5a53ad40a1e65be1622e036c0f6b8/kratos/utilities/brute_force_point_locator.cpp#L10

In your case you could be main author, saying that your work is based on the other process

Reply:
regarding the assertAlmostEqual I have to think
Opinions @rubenzorrilla @roigcarlo @loumalouomega @RiccardoRossi ?

Reply:
regarding the authorship, you can put just yourself.

regarding the asserAlmostEqual, these are reproducibility tests, so the reference value should be VERY close to the value you get. This means using the value you get in a  "reference run" which you consider correct rather than the analytically expected value. Hence we should use a very low tolerance (1e-12?) to ensure that no unwanted changes happen over time

Reply:
So, as I understand, I should store curvature value for some specific nodes and use assertAlmostEqual for them. It will be done.

Gid output cannot be kept too (maybe commented) for later checks?